src/middleware/validator/generic.validator.ts

```diff
@@ -11,12 +11,27 @@ export function validate(req: Request, res: Response, next: NextFunction) {
   next()
 }
 
+export function asInt(idName: string) {
+  return function (req: Request, res: Response, next: NextFunction) {
+    if (!req.params[idName])
+      return res
+        .status(400)
+        .json(new GenericResponse('Missing ' + idName + ' param on ' + idName + ' required request'))
```

Comment:
This is a js thing. Js allows for string templating which is internally a bit faster to run, and a bit cleaner to read 

```suggestion
        .json(new GenericResponse(`Missing  ${idName} param on ${idName} required request`))
```